apps/vscode/src/package.json

```diff
@@ -37,7 +37,7 @@
     "Other"
   ],
   "activationEvents": [
-    "*"
+    "workspaceContains:nx.json"
```

Comment:
I also made this only activate when you have a nx.json file in your vscode workspace. This should free up memory for folks who use vscode for things that aren't Nx 👀